src/js/auth/initializeUserInDb.ts

```diff
@@ -16,10 +16,13 @@ interface InitializeUserInDBParams extends UpdateUsernameInput {
 
 const serverClient = getClient()
 
+/**
+ * Look up in our db (not Auth0) to see whether a user by uuid exists.  If it doesn't then insert a new user profile.
+ */
 export const initializeUserInDB = async (params: InitializeUserInDBParams): Promise<boolean> => {
   const { auth0UserId, accessToken, userUuid, username, email, avatar } = params
-  const existed = await doesUserExist(username)
-  if (existed) {
```

Comment:
`existed` will be false if the user has changed their username to something other than the initial default in Auth0. This causes their new username to be set to the default.